src/pages/api/v1/prep-yatra/recruiter/index.ts

```diff
@@ -0,0 +1,91 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+import { apiStatusCodes } from '@/constant';
+import { addRecruiterToDB } from '@/database';
+import { connectDB } from '@/middlewares';
+import { sendAPIResponse } from '@/utils';
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  await connectDB();
+
+  switch(req.method){
+    case 'POST':
+        return handleAddRecruiter(req,res);
+    default:
+      return res.status(apiStatusCodes.BAD_REQUEST).json(
+        sendAPIResponse({
+          status: false,
+          message: `Method ${req.method} Not Allowed`,
+        })
+      );
+  }
+};
+
+const handleAddRecruiter = async (req: NextApiRequest,res: NextApiResponse) => {
+     try {
+    const {
+      userId,
+      recruiterName,
+      contact,
+      company,
+      appliedPosition,
+      applicationStatus,
+      lastContacted,
```

Comment:
check which field are required and which are not in PY codebase. We can't have everything required. Makes it hard for user to log